### ISSUE TITLE
Need to import requests

### ISSUE THREAD

**Issue by camipacifici** [spacetelescope/astrocut#125]
Hi Folks,
I am installing astrocut using pip in a fresh conda environment with python=3.11. The version it is pulling is 0.11.1.
When I run
`from astrocut import fits_cut`
it complains that it does not find the package `requests`.
If I manually install the `requests` package, everything works. 
I did not have this problem until a month or so ago. Am I missing something? Should `requests` be pulled automatically? Or is this by design? I am not finding any relevant info in the docs, sorry.
Thank you for any explanation!

**Comment by scfleming** [comment]
Maybe need to add this under the setup.cfg file as an "install requires" @snbianco ?

**Comment by snbianco** [comment]
Yep, I will make that change. TIL that `requests` is not part of the standard library!
